File: server/models/product.js
const mongoose = require('mongoose');

const productSchema = new mongoose.Schema({
  title: {
    type: String
  },
  price: {
    type: Number
  },
  description: {
    type: String
  },
  specification: {
    size: {
      width: {
        type: Number
      },
      height: {
        type: Number
      },
    },
    type: {
      type: String
    },
    material: {
      type: String
    },
  },
  tags: {
    type: [String]
  }
});

module.exports = mongoose.model('Product', productSchema);

File: client/app/src/main/java/com/tkpd/hackathon17/data/model/Product.kt
package com.tkpd.hackathon17.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Product (
    val id: String? = null,
    val title: String? = null,
    val price: Float? = null,
    val description: String? = null,
    val tags: List<String>? = null,
    val image: String? = null,
    val exif: Boolean = false,
    val specification: Specification? = null,
) : Parcelable